src/ui/components/Library/Team/View/shared/Alert.tsx

```diff
@@ -0,0 +1,17 @@
+const AlertList = {
+  MISSING_REPLAYS_FOR_TEST_RUN:
+    "No replays were found for this run. They may be outside the retention window or may not have been uploaded",
+  MISSING_REPLAYS_FOR_TEST:
+    "This test has executions that are not reflected in the list of replays below",
+};
+
+export function Alert({ reason, link }: { reason: keyof typeof AlertList; link?: string }) {
+  return (
+    <div data-test-id={reason} className="gap-3 rounded-lg bg-chrome p-3">
+      <span>{AlertList[reason]}</span>
+      <a href={link} rel="noreferrer" target="_blank" className="ml-1 underline">
```

Comment:
We should not show "Learn more" if we don't have a `link`.

Reply:
Good call. I missed this!

Reply:
`link` is provided everywhere this is used so I will mark `link` as required. I think it makes sense to always have additional docs for these warnings so this will enforce that.